<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>181</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>666</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>666</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PLAY |&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PAUSE ||</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>481</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>481</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>%v / %m</string>
    </property>
   </widget>
   <widget class="QSlider" name="ShuttleSlider">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>31</height>
     </size>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="VolumeSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>41</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>91</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>91</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="RepeatOnceRadioButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>111</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REPEAT ONCE</string>
    </property>
   </widget>
   <widget class="QLabel" name="shuttleArrowsLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;                                                         &gt;&gt;&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShuffleCheckbox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SHUFFLE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RepeatRadioButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>71</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REPEAT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PreviousButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt; PREVIOUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NEXT &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="fastForwardLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FAST-FORWARD</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>47</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>47</width>
      <height>14</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VOLUME</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumePlusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Symbol</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeMinusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Symbol</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
   </widget>
   <widget class="QLabel" name="rewindLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REWIND</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_quit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
